proto/cacheclient.proto

```diff
@@ -58,53 +58,66 @@ message _SetResponse {
 
 message _DictionaryGetRequest {
   bytes dictionary_name = 1;
-  repeated bytes dictionary_keys = 2;
+  repeated bytes fields = 2;
 }
 
 message _DictionaryGetResponse {
   message _DictionaryGetResponsePart {
     ECacheResult result = 1;
     bytes cache_body = 2;
   }
-  repeated _DictionaryGetResponsePart dictionary_body = 1;
-  string message = 2;
+
+  message _Found {
+    repeated _DictionaryGetResponsePart items = 1;
+  }
+
+  message _Missing {}
+
+  oneof dictionary {
+    _Found found = 1;
+    _Missing missing = 2;
+  }
 }
 
-message _DictionaryGetAllRequest {
+message _DictionaryFetchRequest {
   bytes dictionary_name = 1;
 }
 
-message _DictionaryKeyValuePair {
-  bytes key = 1;
+message _DictionaryFieldValuePair {
+  bytes field = 1;
   bytes value = 2;
 }
 
-message _DictionaryGetAllResponse {
-  ECacheResult result = 1;
-  repeated _DictionaryKeyValuePair dictionary_body = 2;
-  string message = 3;
+message _DictionaryFetchResponse {
+  message _Found {
+    repeated _DictionaryFieldValuePair items = 1;
+  }
+
+  message _Missing {}
+
+  oneof dictionary {
+    _Found found = 1;
+    _Missing missing = 2;
+  }
 }
 
 message _DictionarySetRequest {
   bytes dictionary_name = 1;
-  repeated _DictionaryKeyValuePair dictionary_body = 2;
+  repeated _DictionaryFieldValuePair items = 2;
   uint64 ttl_milliseconds = 3;
   bool refresh_ttl = 4;
 }
 
-message _DictionarySetResponse {
-  ECacheResult result = 1;
-  string message = 2;
-}
+message _DictionarySetResponse {}
 
 message _DictionaryDeleteRequest {
   message Some {
-    repeated bytes keys = 1;
+    repeated bytes fields = 1;
   }
   message All { }
 
   bytes dictionary_name = 1;
-  oneof delete_keys {
+  oneof delete_fields {
```

Comment:
nit: we can call this `delete` to be consistent with the set and list apis. We use `difference` in SetDifferenceRequest and for list apis also I have used `erase` in ListEraseRequest and `remove` in ListRemoveRequest. 

https://github.com/momentohq/client_protos/pull/71/files

Reply:
Fixed in a332286